sips/sip-120.md

```diff
@@ -1,7 +1,7 @@
 ---
 sip: 120
-title: Keep3r TWAP Exchange Function
-status: Approved
+title: TWAP Exchange Function
+status: Proposed
```

Comment:
@kaleb-keny mentioned that we should probably re-discuss this SIP due to the large changes involved from prior.